src/main/python/paragraph_indexing/README.md

```diff
@@ -45,7 +45,7 @@ Suppose you're under Anserini directory. First indexing
 
 ```bash
 nohup sh target/appassembler/bin/IndexCollection -collection TrecCollection \
- -input /path/to/disk45/ -generator JsoupGenerator \
+ -input /path/to/disk45/ -generator DefaultLuceneDocumentGenerator \
```

Comment:
This is really old code, I think we can delete, period?

Seems like `passage_retrieval/example/robust04.md` is the updated version?